src/core-services/core/util/loadSampleData.js

```diff
@@ -44,7 +41,7 @@ export default async function loadSampleData(context) {
       throw new Error("More than one primary shop in the sample dataset");
     }
 
-    const currentDomain = rootUrl && url.parse(rootUrl).host;
+    const currentDomain = rootUrl && new URL(rootUrl).host;
```

Comment:
Most places are looking up shop by domain using `hostname` but here we set it based on `host`. One includes the port and one does not. Should the other places be changed to `host`? I'm thinking whichever one doesn't have port is probably correct for this situation.

Reply:
Looks like `hostname` does not have port: https://nodejs.org/api/url.html#url_url

Updated